.github/workflows/clear-environment.yml

```diff
@@ -22,6 +22,8 @@ jobs:
     name: 'Reset data'
     environment: ${{ github.event.inputs.environment }}
     runs-on: ubuntu-22.04
+    outputs:
+      outcome: ${{ steps.reset-data.outcome }}
```

Comment:
Did you test this works? I was expecting to see a step with `id: reset-data` 🤔

Reply:
I hadn't tested it 😅 . You're absolutely right, I forgot to add in the id